app/utils/constants.js

```diff
@@ -8,3 +8,26 @@ export const Keyboard = {
   UP: 38,
   DOWN: 40
 };
+
+export const ROLES = {
+  member: 'Medlem',
+  leader: 'Leder',
+  treasurer: 'Økonomiansvarlig',
+  'co-leader': 'Nestleder',
+  active_retiree: 'Aktiv pang',
+  retiree: 'Pang',
+  recruiting: 'Rekruttering',
+  development: 'Utvikling',
+  editor: 'Moderator',
+  media_relations: 'PR',
+  alumni: 'Alumni',
+  webmaster: 'Web',
+  interest_group_admin: 'Admin',
```

Comment:
maybe remove these admin roles and rather do something like `ROLES[role] || _.startCase(role)` instead? which would show `Trip Admin`